build-system/babel-plugins/babel-plugin-transform-promise-resolve/index.js

```diff
@@ -17,6 +17,17 @@
 const pathmodule = require('path');
 const {addNamed} = require('@babel/helper-module-imports');
 
+/**
+ * @param {*} babel
+ * @param {{
+ *  importFrom?: string,
+ * }=} options
+ * @return {{
+ *   visitor: {
+ * CallExpression: {Funcion(path: string): void}
+ *  }
+ * }}
+ */
```

Comment:
General question for folks like @jridgewell, @samouri, and @erwinmombay who add / maintain most of our custom babel plugins. Is it too much of a tax to pay to keep these types accurate?

In a future PR, @rileyajones is going to make this type info a requirement, similar to runtime code. On the plus side, he has done the hard work of adding all missing types before enforcing the rule 😃

Reply:
They aren't so frequently updated, and keeping types correct is always a worthwhile trade